src/oemof/solph/_energy_system.py

```diff
@@ -130,13 +130,6 @@ def __init__(
                     "conflicting to each other."
                 )
                 raise AttributeError(msg)
-            else:
-                msg = (
-                    "Ensure that your timeindex and timeincrement are "
-                    "consistent.\nIf you are not considering non-equidistant "
-                    "timeindices, consider only specifying a timeindex."
-                )
-                warnings.warn(msg, debugging.SuspiciousUsageWarning)
```

Comment:
I cannot say things about this, because I am not really knowledgable where this comes (or came) into place.

Reply:
I deleted the message as the warning only showed up when using experimental features anyway. I think, @jokochems should know best if it should show up. (If so, it should be an `ExperimentalFeatureWarning`.)

Reply:
Yes, I think it is important to be aware of this when setting up a multi-period model. Also, I think the second sentence does not hold as @henhuy found that both needs to be explicit currently (see #979). I'm fine with transferring it to an `ExperimentalFeatureWarning`, but would vote against a removal.

Reply:
> Yes, I think it is important to be aware of this when setting up a multi-period model. Also, I think the second sentence does not hold as @henhuy found that both needs to be explicit currently (see #979). I'm fine with transferring it to an `ExperimentalFeatureWarning`, but would vote against a removal.

Of course, the desired longer-term fix would be to properly address and resolve #979.

Reply:
I re-introduced the warning, now without the second sentence and as a `ExperimentalFeatureWarning`.